src/bootstrap/bin/rustdoc.rs

```diff
@@ -41,7 +41,12 @@ fn main() {
         cmd.arg(arg);
     }
     if env::var_os("RUSTDOC_FUSE_LD_LLD").is_some() {
-        cmd.arg("-Clink-args=-fuse-ld=lld");
+        cmd.arg("-Clink-arg=-fuse-ld=lld");
+        if cfg!(windows) {
```

Comment:
This should check for `windows-msvc` because `windows-gnu` uses the same syntax as Unix.